docs/docs/transforms/partition.md

```diff
@@ -21,7 +21,7 @@ Internally, this transform processes a collection of special tree node objects g
 | padding             | {% include type t="Number" %}  | The padding to include between adjacent nodes (default `0`).|
 | round               | {% include type t="Boolean" %} | Indicates if node layout values should be rounded (default `false`).|
 | size                | {% include type t="Number[]" %}| The size of the layout, provided as a [width, height] array.|
-| as                  | {% include type t="String[]" %}| The output fields at which to write the layout results. The default is `["x0", "y0", "x1", "y1", "depth", "children"]`, where `x0`, `y0`, `x1` and `y1` are the starting and ending layout coordinates for each segment, `depth` is the tree depth, and `children` is the count of a node's children in the tree.|
+| as                  | {% include type t="String[]" %}| The output fields at which to write the layout results. The default is `["x0", "y0", "x1", "y1", "depth", "value", "children"]`, where `x0`, `y0`, `x1` and `y1` are the starting and ending layout coordinates for each segment, `depth` is the tree depth, `children` is the count of a node's children in the tree, and `value` is the sum of values for a node and all its descendants.|
```

Comment:
Typo. The order of field descriptions is incorrect. Based on the default list, `value` should come before `children`. However, I think the actual order should be `children`, then the new `value` field, in order to avoid introducing a breaking change.